src/themes/elife/styles.css

```diff
@@ -148,14 +161,94 @@ main {
   }
 }
 
-:--datePublished {
-  color: var(--color-text-secondary);
-  display: block;
+:--references {
   font-family: 'Noto Sans', Arial, Helvetica, sans-serif;
-  font-size: calc(var(--text-size-rem-small) * 1rem);
-  letter-spacing: var(--letter-spacing);
-  line-height: calc(var(--baseline-measure) / var(--text-size-rem-small));
-  text-transform: capitalize;
+
+  ol {
+    counter-reset: item;
+    list-style-type: none;
+    margin-left: 1.5rem;
+
+    li[itemprop='citation'] {
+      display: block;
+      font-family: 'Noto Sans', Arial, Helvetica, sans-serif;
+      font-weight: bold;
```

Comment:
This is causing a lot of overriding further down. Better to remove from here and only apply it where you need it.